forest/sycamore/common.py

```diff
@@ -398,6 +399,10 @@ def aggregate_surveys_config(
             The first date of the survey data, in YYYY-MM-DD format
         time_end(str):
             The last date of the survey data, in YYYY-MM-DD format
+        history_path: Path to survey history file. If this is included, the
```

Comment:
Either style is acceptable but you should continue putting arg names on a separate line for consistency.

Reply:
Ah, didn't realize I was doing that. Thanks!